start.sh

```diff
@@ -42,6 +42,20 @@ prep() {
     ${COMPOSE} pull
 }
 
+# Gets the IP of the local machine.
+getIp() {
```

Comment:
Given that we've got something called `getIpPort` below, maybe we should call this `getLocalIp` or something?